string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,22 @@
+/* exported getWords */
+function getWords(string) {
+  if (string.length === 0) {
+    return string.split('');
```

Comment:
Rather than using the split method to get an empty array, you can just return an empty array literal directly `[]`